homework2/task2.js

```diff
@@ -0,0 +1,16 @@
+function Check(str){
+    let splited = str.split("")
+    
+    let result = 0;
+  
+    for ( let i=0 ; i < splited.length;i++){
+      if(splited[0] === "i" && splited[1] === "f"){
+        result = true
+      }else{
+        result = false
+      }
+      console.log(result)
+    }
+  }
+  Check("")
```

Comment:
![task2](https://user-images.githubusercontent.com/61050698/199193971-96b5b436-a0c1-445c-a8ee-1f14f0a0e24e.png)

Reply:
The program should return result only once

Reply:
Fixed